extra/utils/Permission.js

```diff
@@ -0,0 +1,40 @@
+const permissions =
+{
+    'getUsers': {
+        all: ['head-trainer'],
+        read: ['trainer'],
+        write: ['trainee'],
+        Delete: [],
+    }
+}
+function hasPermission(moduleName, role, permissionType) {
+    const { all, read, write, Delete } = moduleName;
+    console.log("--------",moduleName);
+    // const temp = permissions[moduleName];
+    // console.log("--------",temp);
+    let type;
+    if (type = all.includes(role))
+    {
+        return true;
+    }
+    else{
+        if (permissionType == 'read'){
+            type = read.includes(role)
+            return type;
+        }
+        else if(permissionType=="write"){
+            type = write.includes(role);
+            return type;
+        }
+        else if(permissionType=="Delete"){
+            type = Delete.includes(role)
+            return type;
+        }
+    }
+}
+let result = hasPermission(permissions.getUsers,"head-trainer","all");
+console.log(result);
+let result_1 = hasPermission(permissions.getUsers,"trainee","read");
+console.log(result_1);
```

Comment:
Need to pass the moduleName only not module itself

Reply:
ya i passsed the moduleName and it running correctly